ydb/core/kqp/provider/yql_kikimr_exec.cpp

```diff
@@ -1922,6 +1890,37 @@ class TKiSinkCallableExecutionTransformer : public TAsyncCallbackTransformer<TKi
             }, "Executing CREATE SEQUENCE");
         }
 
+        if (auto maybeCreate = TMaybeNode<TKiCreateTopic>(input)) {
+            auto requireStatus = RequireChild(*input, 0);
+            if (requireStatus.Level != TStatus::Ok) {
+                return SyncStatus(requireStatus);
+            }
+            auto cluster = TString(maybeCreate.Cast().DataSink().Cluster());
+            TString topicName = TString(maybeCreate.Cast().Topic());
+            Ydb::Topic::CreateTopicRequest createReq;
+            createReq.set_path(topicName);
+            for (const auto& consumer : maybeCreate.Cast().Consumers()) {
+                auto error = AddConsumerToTopicRequest(createReq.add_consumers(), consumer);
+                if (!error.empty()) {
+                    ctx.AddError(TIssue(ctx.GetPosition(input->Pos()), TStringBuilder() << error << input->Content()));
+                    return SyncError();
+                }
+            }
+            AddTopicSettingsToRequest(&createReq,maybeCreate.Cast().TopicSettings());
+            bool existingOk = (maybeCreate.ExistingOk().Cast().Value() == "1");
+
+            // DEBUG
+            // Cerr << "Create topic request proto: " << createReq.DebugString() << Endl;
+            auto future = Gateway->CreateTopic(cluster, std::move(createReq), existingOk);
```

Comment:
existingOk is ignored inside. Is this correct?

Reply:
Classic DDL doesn't support it, so it's ignored. Could throw an error here insted when IF NOT EXISTS is provided though.

Reply:
Throwing an error sounds good.